vue-frontend/src/components/android-app-development/SuccessStorySection.vue

```diff
@@ -0,0 +1,296 @@
+                    >
+                      {{ getFirstWord(client.name) }}</span
+                    >
+                  </div>
+                </swiper-slide></swiper
+              >
+            </div>
+
+            <swiper
+              :slidesPerView="1"
+              :centeredSlides="true"
+              :spaceBetween="10"
+              :direction="'vertical'"
+              :autoplay="{
+                delay: 4000,
+                disableOnInteraction: false,
+              }"
+              :loop="true"
+              :loopedSlides="50"
+              class="swiper-container tw-h-[600px] lg:tw-h-[700px] tw-flex tw-flex"
+              @swiper="setSwiperRef"
+              @slideChange="onSlideChange"
+            >
+              <swiper-slide
+                v-for="(client, index) in clients"
+                :key="index"
+                class="tw-cursor-pointer xl:tw-p-12 tw-flex tw-justify-center tw-flex-col"
+              >
+                <div
+                  class="tw-ml-[-1rem] md:tw-pr-6 lg:tw-px-4 xl:tw-px-24"
+                ></div>
+                <div class="tw-flex tw-gap-x-4 tw-my-5">
+                  <span class="tw-flex tw-flex-row">
+                    <font-awesome-icon
+                      v-for="i in 5"
+                      :key="i"
+                      class="fa-star tw-w-[1.5rem] tw-h-[1.5rem] footer-icon"
+                      icon="star"
+                  /></span>
+                  <span
+                    class="tw-font-inter-regular tw-text-[1.125rem] tw-leading-[1.6875rem] tw-text-black-core/[0.87]"
+                    >(5.0)</span
+                  >
+                </div>
+                <div
+                  class="tw-font-inter-semibold tw-text-[1.5rem] lg:tw-text-[2.125rem] tw-leading-[2.5rem] lg:tw-leading-[3.1875rem]"
+                  v-html="client.review"
+                ></div>
+                <div class="tw-flex tw-flex-row tw-justify-between tw-my-5">
+                  <div class="tw-flex tw-flex-col">
+                    <span
+                      class="tw-mt-3 v2-canopas-gradient-text tw-font-inter-semibold tw-text-[1.25rem] tw-leading-[1.5rem]"
+                      >{{ client.name }}</span
+                    >
+                    <span
+                      class="tw-mt-4 tw-font-inter-regular tw-text-[1.125rem] tw-leading-[1.35rem] tw-text-black-core/[0.60]"
+                      v-html="client.desktop_designation"
+                    ></span>
+                  </div>
+                </div>
+              </swiper-slide>
+            </swiper>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import SwiperCore, { Autoplay } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import usericon from "@/assets/images/andriod-app-development/icon.webp";
```

Comment:
its not looking like figma's icon

Reply:
digita told me to prefer link website. as figma don't support many animation

Reply:
i am talking about icon